src/sentry/models/sentryshot.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import uuid
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import Model
+from sentry.db.models.base import region_silo_only_model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_only_model
+class SentryShot(Model):
```

Comment:
What's the plan for cleaning up old snapshots?

Reply:
I plan to set up a TTL for 30 days.

Reply:
Ok, probably worth adding an index on `date_added` to facilitate that too.

Reply:
This model should eventually be added to `sentry.runner.commands.cleanup` so that existing retention cleanup can purge it.